it/src/main/scala/quasar/physical/s3/LWCTests.scala

```diff
@@ -0,0 +1,291 @@
+/*
+ * Copyright 2014–2018 SlamData Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package quasar.physical.s3
+
+import slamdata.Predef._
+
+import pathy._
+import quasar.Data
+import quasar.contrib.pathy._
+import quasar.fs.mount.ConnectionUri
+import quasar.mimir.LightweightFileSystem
+import scalaz._, Scalaz._
+import scalaz.concurrent.Task
+
+final case class TestFS(
```

Comment:
Is there a reason you have this file in `it/src/main/` rather than `it/src/test/`?

Reply:
Also curious why you designed this with a `main` method, and not using `specs2`?

Reply:
I've basically answered these two questions for myself after thinking about it a little more :)